src/psyclone/domain/lfric/kernel_interface.py

```diff
@@ -368,6 +382,7 @@ def scalar(self, scalar_arg, var_accesses=None):
                 scalar_arg.name, mapping[scalar_arg.intrinsic_type],
                 interface=ArgumentInterface(INTENT_MAPPING[scalar_arg.intent]))
         except KeyError:
+            # pylint: disable=raise-missing-from
```

Comment:
My pylint says "Bad option value 'raise-missing-from'"

Reply:
Get a newer version of pylint? ;-) Mine is 2.6.0. The errors that I get without the directives are

```
domain/lfric/kernel_interface.py:91:8: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
domain/lfric/kernel_interface.py:105:8: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
domain/lfric/kernel_interface.py:197:12: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
```

I've removed them in any case.

Reply:
There was no need to remove them! I was (wrongly) assuming that you'd be running an older version of pylint that me :-) I've upgraded my pylint.